tests/runtime/probe

```diff
@@ -76,6 +76,23 @@ EXPECT Offset outside the function bounds \('vfs_read' size is*
 TIMEOUT 5
 WILL_FAIL
 
+NAME kprobe_offset_module
```

Comment:
Maybe add comment that this test relies on lack of module instruction checking support?

Reply:
@danobi, it's just punting the check to the kernel - if you get the offset wrong, the kernel won't let the program attach. After all, if we implemented module disassembly and offset checking, this test would _still_ pass, so it's clearly not relying on the absence of anything.